packages/sdk/src/dataProtector/sharing/subgraph/getCollectionById.ts

```diff
@@ -0,0 +1,38 @@
+import { gql, type GraphQLClient } from 'graphql-request';
+import type { Address } from '../../types/index.js';
+
+export async function getCollectionById({
+  graphQLClient,
+  collectionTokenId,
+}: {
+  graphQLClient: GraphQLClient;
+  collectionTokenId: Address;
+}) {
+  const getProtectedDataQuery = gql`
+    query {
+      collection(id: "${collectionTokenId}") {
+        id
+        owner {
+          id
+        }
+        protectedDatas {
+          id
+        }
+        subscriptionParams {
+          duration
+          price
+        }
+      }
+    }
+  `;
+  const { collection } = await graphQLClient.request<{
+    collection: {
+      id: Address;
+      owner: { id: Address };
+      protectedDatas: { id: Address }[];
+      subscriptionParams: { duration: number; price: number };
+      subscriptions: { endDate: number }[];
```

Comment:
Not present in the query at the moment. Add it?

Or remove it from here as it seems it's not used anywhere so far.

Reply:
411db3ec14e3f4ae81d700ea12585d30ca935b42